src/foraging_gui/Foraging.py

```diff
@@ -520,6 +521,17 @@ def _GetSettings(self):
         self.newscale_port_tower2=Settings['newscale_port_tower2']
         self.newscale_port_tower3=Settings['newscale_port_tower3']
         self.newscale_port_tower4=Settings['newscale_port_tower4']
+        
+        # Also stream log info to the console if enabled
```

Comment:
Minor - I would call it `show_log_info_in_console`, since the "enable" feels redundant with the setting being True/False

Reply:
fixed.